site3/website/src/pages/release-notes.md

```diff
@@ -1,6 +1,99 @@
 <!-- markdown-link-check-disable -->
 # Release notes
 
+## 4.17.0
+
+Release 4.17.0 includes multiple important features, improvements, bug fixes and some dependencies CVE fixes.
+
+The technical details of this release are summarized below.
+
+### Breaking Changes
+No breaking changes. Some defaults are different, but overall there are no compatibility concerns.
+
+### Features
+* BP-62 Batch Read API
```

Comment:
Do we need to add the detailed PRs of batch read?

Reply:
it is not useful for end users